src/pages/content-edit.js

```diff
@@ -70,6 +70,17 @@ module.exports = {
                         class   : css.publishing,
                         enabled : ctrl.form && ctrl.form.checkValidity()
                     }),
+                    m("div", { class : css.actions }, [
+                        m("a", {
+                                title  : "Preview",
+                                href   : "/content/" + ctrl.schema.preview + m.route.param("id"),
```

Comment:
This preview link should go to the same place as https://github.com/tivac/crucible/pull/36/files#diff-968046e1d94760600143870640c14520R199 - it's so while you're editing content you can open a new tab that shows the current content in its external site. Previewing it within crucible from here doesn't add much value because it just looks like the page you're already on.